demisto_sdk/commands/upload/uploader.py

```diff
@@ -50,7 +50,7 @@
 
     FileType.INCIDENT_TYPE,
     FileType.INCIDENT_FIELD,
-    # FileType.REPUTATION,  currently not supported by demisto-py
+    FileType.REPUTATION,
```

Comment:
This is still not supported upload as an entity,
Upload Pack is another endpoint.
Am I right @yaakovi ?

Reply:
Checked with Yaakovi and you are right, reverting it.